lib/uaa/cli/curl.rb

```diff
@@ -27,9 +27,10 @@ class CurlCli < CommonCli
     define_option :data, "-d", "--data <data>", "data included in request body"
     define_option :header, "-H", "--header <header>", "header to be included in the request"
     define_option :insecure, "-k", "--insecure", "makes request without verifying SSL certificates"
+    define_option :cacert, "-C", "--cacert <ca_file>", "CA certificate to verify peer against"
```

Comment:
uaac target allows --ca-cert
uaac curl allows --cacert

I know that it is different, but in curl it is --cacert , so should it be the same ?